draft-ietf-quic-tls.md

```diff
@@ -328,8 +328,7 @@ encryption levels:
 
 - PADDING and PING frames MAY appear in packets of any encryption level.
 
-- CRYPTO frames and CONNECTION_CLOSE frames signaling errors at the QUIC layer
-  (type 0x1c) MAY appear in packets of any encryption level except 0-RTT.
+- CRYPTO frames MAY appear in packets of any encryption level except 0-RTT.
 
 - CONNECTION_CLOSE frames signaling application errors (type 0x1d) MUST only be
   sent in packets at the 1-RTT encryption level.
```

Comment:
For consistency, should we strike this point to allow the use of application-level CONNECTION_CLOSE frames in 0-RTT? Looking at the changes proposed for the transport draft, I believe that that is the intent.